types/express-fileupload/index.d.ts

```diff
@@ -1,12 +1,14 @@
-// Type definitions for express-fileupload 1.1
+// Type definitions for express-fileupload 1.2
 // Project: https://github.com/richardgirges/express-fileupload#readme
 // Definitions by: Gintautas Miselis <https://github.com/Naktibalda>
 //                 Sefa Ilkimen <https://github.com/silkimen>
 //                 Tomas Vosicky <https://github.com/vosatom>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 Mark Oude Elberink <https://github.com/markxoe>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import express = require('express');
+import busboy = require('busboy');
```

Comment:
please use `import type { BusboyConfig } `
DT is now on 3.8, so one can point actual intent (import type, not actual symbols),

thx!

Reply:
Is `import type { BusboyConfig } from 'busboy';`
or `import type busboy = require('busboy');` better in this case?

Reply:
Guess I'll go with
```
import type { RequestHandler } from 'express';
import type { BusboyConfig } from 'busboy';
```
just to enforce code style